src/MSBuild/app.amd64.config

```diff
@@ -46,18 +46,6 @@
           <codeBase version="15.1.0.0" href="..\Microsoft.Build.Conversion.Core.dll"/>
         </dependentAssembly>
 
-        <!-- Redirects for facade assemblies -->
```

Comment:
Do we really not need these any more? I think I expected them to remain (maybe with an updated version) with no `codeBase`.

Reply:
Very good question. Those types moved in-box, so I figured that if VS also moves to 472, they'll also start using the in-box types, in which case the redirects are not needed anymore. Is this a valid assumption?